logging/settable/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 Piotr Tabor. All Rights Reserved.
```

Comment:
I'm not sure this is an appropriate license header for this project, could you please remove it?

Reply:
Removed. Based this on `logging/doc.go` style.